### ISSUE TITLE
Links to GitHub repo at the top of the site are broken

### ISSUE THREAD

**Issue by alimanfoo** [anopheles-genomic-surveillance/selection-atlas#117]
Think it has been configured as "malaria-vector-selection-atlas" but should be "selection-atlas".

**Comment by alimanfoo** [comment]
Resolved via #124
